draft-ietf-tls-tls13.md

```diff
@@ -1517,7 +1517,7 @@ Structure of this message:
            Random random;
            opaque legacy_session_id<0..32>;
            CipherSuite cipher_suites<2..2^16-2>;
-           opaque legacy_compression_methods<1..2^8-1>;
+           opaque legacy_compression_methods[2] = 0x0100;
```

Comment:
I considered doing this to legacy_session_id as well, but the understanding is that you might generate a TLS 1.3 ClientHello when resuming a TLS 1.2 sesssion.